Fix typos in debug logs

barpavel opened the pull request:
Minor typos seen during the previous Code Review.

abawer committed:
readme: Add prerequisites section

Signed-off-by: Amit Bawer <abawer@redhat.com>

abawer committed:
tox: Add python version as make argument

tox -e py27 will invoke
make PY_VERSION=2.7

tox -e py36 will invoke
make PY_VERSION=3.6

At this point rpm spec invokes
make PY_VERSION=2.7

co-authored with Pavel Bar <pbar@redhat.com>

Signed-off-by: Amit Bawer <abawer@redhat.com>

abawer committed:
tests: Use python3 and python2 compatible socket exception type

Replace socket.error with EnvironmentError exception class

co-authored with Nir Soffer <nsoffer@redhat.com>

Signed-off-by: Amit Bawer <abawer@redhat.com>

abawer committed:
tests: Lockspace and resource names should be read as bytes

This patch is preparation for python 3 porting. No regression for tests
under python 2 which is agnostic to strings/bytes.

Signed-off-by: Amit Bawer <abawer@redhat.com>

Signed-off-by: Amit Bawer <abawer.com@gmail.com>

abawer committed:
tests: Enforce absolute import behavior for python2 tests

added from __future__ import absolute_import

at top of tests modules.

co-authored with Nir Soffer <nsoffer@redhat.com>

Signed-off-by: Amit Bawer <abawer@redhat.com>

aabawer committed:
sanlock: Resource API should only support disks as a tuple

As stated in sanlock module help:
The disks must be in the format: [(path, offset), ... ]

Signed-off-by: Amit Bawer <abawer@redhat.com>

aabawer committed:
tests: Test parsing a resource which is not a list of tuples

Expecting to get ValueError for resource argument which
is not a list of tuples (disk, offset).

Signed-off-by: Amit Bawer <abawer@redhat.com>

aabawer committed:
sanlock: Provide internal common module init function

This will serve python 3 porting in future patches to come.

Signed-off-by: Amit Bawer <abawer@redhat.com>

barpavel committed:
Adding constants for Kibibyte, Mebibyte, Gibibyte, Tebibyte & Pebibyte and using them everywhere in tests.

Signed-off-by: Pavel Bar <pbar@redhat.com>

nirs reviewed (approved):
Looks good, please rebase on master.

nirs commented:
Pleased don't mix unrelated commits. Every PR should be based on master so we can merge it easily in whatever order.

Please take the constants commit out of this PR.

nirs committed:
tests: Update lockspace and resource tests

We introduced new assertions about lockspace and resource names in the
4k branch. Update the assertions to check for b"name" instead of "name".

Signed-off-by: Nir Soffer <nsoffer@redhat.com>

nirs reviewed (changes requested):
Please remove the constants change from this patch

nirs commented:
Already fixed in #9
